docs/admin/hardware.rst

```diff
@@ -35,7 +35,32 @@ The ThinkPad T14 **with a 10th-generation Intel Core processor** is a recommende
 
 - Otherwise, follow the instructions below to ensure that the BIOS is up to date.
 
-The Ethernet controller will not immediately work out of the box and require a one-time manual configuration on install. After Qubes starts for the first time, when ``sys-net`` fails to start, follow the instructions below for the :ref:`thinkpad_t490`, but only for the ``dom0:00_1f.6`` Ethernet device.
+The Ethernet and Wi-Fi controllers will not work without one-time manual configuration, as documented in the following sections.
+
+Ethernet controller
+^^^^^^^^^^^^^^^^^^^
+After Qubes starts for the first time, when ``sys-net`` fails to start, follow the instructions below for the :ref:`thinkpad_t490`, but only for the ``dom0:00_1f.6`` Ethernet device.
+
+Wi-Fi controller
+^^^^^^^^^^^^^^^^
+Once ``sys-net`` starts, the Intel AX210 Wi-Fi controller will not automatically be available to the Network Manager for you to select a network. [#ax210_dmesg]_  To fix this:
+
+#. Connect the machine to the Internet via Ethernet.
+#. :doc:`Install and update the Fedora 36 template. <upgrading_fedora>`
+#. In the ``sys-net`` VM, run the following command::
+
+      curl https://raw.githubusercontent.com/freedomofpress/securedrop-workstation-docs/main/docs/includes/iwlwifi/rc.local | sudo tee /rw/config/rc.local
+
+#. Then run the following command and verify the expected output::
```

Comment:
I added this verification step on the assumption that it might be beneficial for (Linux-savvy) Qubes/Workstation administrators to take a look at what this workaround is doing in case they need to troubleshoot further.  If we disagree with that assumption, and/or if we want an explicit security verification, we could replace the `cat` command and output with a `sha256sum` command and output, as we do for `securedrop-workstation-dom0-config-<versionNumber>-1.fc32.noarch.rpm` as the only other artifact we currently ask administrators to download manually.  (CC: @l3th3 both as security engineer and as likely reviewer here. :-)

I'll open a follow-up ticket for the broader question of how we want (and when we're willing) to provide hardware-compatibility workarounds and shims.